index.html

```diff
@@ -4188,22 +4188,26 @@ <h2>HTTP</h2>
         </section>
 
         <section class="informative">
-          <h2>JSON Processing</h2>
+          <h2>Credential Type-Specific Processing</h2>
 
           <p>
-While the media types describing conforming documents defined in this
-specification always express JSON-LD, JSON-LD processing is not required to be
-performed, since JSON-LD is JSON. Some scenarios where processing a
-<a>verifiable credential</a> or a <a>verifiable presentation</a> as JSON is
-desirable include, but are not limited to:
+<dfn>General JSON-LD processing</dfn> is defined as a mechanism that utilizes a
+JSON-LD software library to process a <a>conforming document</a> by performing
+various <a data-cite="?JSON-LD11#forms-of-json-ld">transformations</a>.
+<dfn>Credential type-specific processing</dfn> is defined as a lighter-weight
+mechanism, that doesn't require a JSON-LD software library, to process
+<a>conforming documents</a>. General JSON-LD processing is not always required
+to be performed when working with <a>verifiable credentials</a> and
+<a>verifiable presentations</a>; one can use credential-type-specific
+processing, instead. Some scenarios where credential-type-specific processing
+is desirable include, but are not limited to:
```

Comment:
```suggestion
<a>conforming documents</a>. Some consumers of <a>verifiable credentials</a>
only need to consume specific credential types. These consumers can use
credential-type-specific processing instead of generalized processing. Some scenarios
where credential-type-specific processing is desirable include, but are not limited to:
```